scripts/build_site.sh

```diff
@@ -19,7 +19,7 @@ args=$*
 
 # Checking prerequisites for folders
 # Getting the link for github repository
-GITHUB_REMOTE="git@github.com:material-components/material-components-site-generator.git"
+GITHUB_REMOTE="https://github.com/material-components/material-components-site-generator.git"
```

Comment:
Why are we changing the URL as part of this change? Can you document the reason in the commit message, please?

Reply:
Done.